src/relayFeeCalculator/chain-queries/arbitrum.ts

```diff
@@ -0,0 +1,47 @@
+import { QueryInterface } from "../relayFeeCalculator";
+import { BigNumberish } from "../../utils";
+import ethers, { BigNumber } from "ethers";
+import { SymbolMapping } from "./ethereum";
+import { Coingecko } from "../../coingecko/Coingecko";
+import { ArbitrumSpokePool__factory, ArbitrumSpokePool } from "@across-protocol/contracts-v2";
+
+class ArbitrumQueryInterface implements QueryInterface {
+  private spokePool: ArbitrumSpokePool;
+
+  constructor(readonly provider: ethers.providers.Provider) {
+    // TODO: replace with address getter.
+    this.spokePool = ArbitrumSpokePool__factory.connect("0xe1C367e2b576Ac421a9f46C9cC624935730c36aa", provider);
```

Comment:
address shoudl be an argument so we can pass it in from the frontend. this will change depending on if we are running testnets

Reply:
Done.